saas_app/locale/fr/LC_MESSAGES/django.po

```diff
@@ -1173,10 +1238,136 @@ msgstr "Résultats de recherche"
 msgid "No results found"
 msgstr "Aucun résultat trouvé"
 
+#: templates/training/training_request.html:8
+#: templates/training/view_request.html:8
+msgid "Training Form Request"
+msgstr "Demande de formulaire de formation"
+
+#: templates/training/training_request.html:11
+#: templates/training/view_request.html:11
+msgid "Training Request Form"
+msgstr "Demande de formulaire de formation"
```

Comment:
The order probably got scrambled:

It's a "Formulaire de demande de formation". Current syntax would mean: Request for a training form 😄